tools/util.ts

```diff
@@ -38,3 +39,35 @@ function findFilesWalk(paths: string[], depth: number) {
 
   return [].concat(...foundPaths);
 }
+
+export const executableSuffix = platform.os === "win" ? ".exe" : ""
+
+/**
+ * Returns true if the path exists.
+ */
+export function existsSync (path: string): boolean {
+  try {
+    lstatSync(path);
+  } catch (e) {
+    return false
+  }
+  return true
+}
+
+/**
+ * Looks up the available deno path with the priority
+ * of release -> debug -> global
+ */
+export function lookupDenoPath(): string {
```

Comment:
Aside: I wonder if the executable location should be available in deno.

Reply:
I think that's convenient in some situations, like when deno is unavailable in $PATH.